html-relative-links/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="index.html">Home Page</a></li>
+    <li><a href="./about-me/about-me.html">About Me</a></li>
```

Comment:
You dont need the './' before these paths! You can say href="about-me/about-me.html"